parent/pom.xml

```diff
@@ -771,6 +772,18 @@
                 <scope>test</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.tngtech.archunit</groupId>
+                <artifactId>archunit</artifactId>
```

Comment:
should we use test scope for the library?

Reply:
if my understanding is correct, we use parent/pom just to declare possible dependencies and theirs versions. The scope should be defined in the child module this dep will be used

Reply:
Yes, you are right. However the lib is for testing purposes, isn't it? If so, let's say test scope here, otherwise it required to duplicate it for downstream modules